integration-tests/bats/merge.bats

```diff
@@ -456,7 +456,7 @@ SQL
     dolt commit -am "right commit"
```

Comment:
Could you check out the `skip` on line 434? Do we detect that constraint violation yet? Just wondering if we can remove that skip now.